test/gtest/ucp/test_ucp_tag_match.cc

```diff
@@ -18,6 +18,7 @@ class test_ucp_tag_match : public test_ucp_tag {
 
     virtual void init()
     {
+        modify_config("TM_THRESH", "1");
```

Comment:
what is 1 exactly ?

Reply:
We need this to enable hardware tag matching

Reply:
1 - enable , 0 - disable ? 
Is it disabled by default ?

Reply:
1 is the minimal message size for using HW tag-matching